keadm/cmd/keadm/app/cmd/edge/join.go

```diff
@@ -114,6 +114,9 @@ func addJoinOtherFlags(cmd *cobra.Command, joinOptions *types.JoinOptions) {
 
 	cmd.Flags().StringVar(&joinOptions.TarballPath, types.TarballPath, joinOptions.TarballPath,
 		"Use this key to set the temp directory path for KubeEdge tarball, if not exist, download it")
+
+	cmd.Flags().StringVarP(&joinOptions.Labels, types.Labels, "l", joinOptions.Labels,
+		`use this key to set the customized labels for node. you can input customized labels like {"key1":"value1", "key2":"value2"}`)
```

Comment:
I recommend using StringSliceVarP to resolve the label flag, like `-l key1=value1,key2=value2`.
What do you think?

Reply:
lgtm,thanks for your suggesstion